Add a contribution guide

chasenlehara opened the pull request:
- Move the current contribution info from the README into the guide
- Add a Code of Conduct
- Add a CONTRIBUTING.md file to the root of the repo that points to the website
- Add the contribution guide to the Guides dropdown & page
- Add the presentations list from https://github.com/bitovi/bitovi/blob/master/great-technology/evangelism/presentations.md

Fixes #295

chasenlehara committed:
Add a contribution guide

- Move the current contribution info from the README into the guide
- Add a Code of Conduct
- Add a CONTRIBUTING.md file to the root of the repo that points to the website
- Add the contribution guide to the Guides dropdown & page
- Add the presentations list from https://github.com/bitovi/bitovi/blob/master/great-technology/evangelism/presentations.md

Fixes #295

matthewp commented on docs/guides/guides.md:
```diff
@@ -61,3 +61,7 @@ Specifically, this guide will walk through the implementation of the following b
  - Setup node services and server-side rendering on the same process.
  - How to turn off parts of the app that should not be server-side rendered.
+
+## [Contributing](./contributing.html)
```
should be able to do `[contributing Contributing]` instead, using the doc name prevents you from having to hardcode the link like that.

chasenlehara replied:
Is that how DocumentJS works or is that only in `bit-docs`? Note that I want to merge this to `master`.

matthewp replied:
They both work this way. if you just do `[contributing]` it would use whether title is given to the contributing docname.

phillipskevin commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,31 @@
+- [Suggesting Features](https://donejs.com/contributing.html#section=section_SuggestingFeatures)
+- [Finding Ways to Contribute](https://donejs.com/contributing.html#section=section_FindingWaystoContribute)
+	- [Finding open issues](#section=section_Findingopenissues)
+	- [Getting involved in the community](#section=section_Gettinginvolvedinthecommunity)
```
is the rest of link intentionally left off?

phillipskevin replied:
and the one above and one below.

chasenlehara replied:
Not intentional, thanks for catching that.

chasenlehara committed:
Fix the links in CONTRIBUTING.MD

Changes based on feedback:

- https://github.com/donejs/donejs/pull/815/files#r101148985
- https://github.com/donejs/donejs/pull/815/files#r101108779

justinbmeyer commented:
where did the contributes page go?  THe diff shows it all in red.

chasenlehara commented:
@justinbmeyer It’s here: https://github.com/donejs/donejs/pull/815/files#diff-54741d26fa44ca37b832eee981ddc15b

The file is so different from what Julia had that git considers it a new file, and it’s so large that GitHub won’t load it by default, so you have to click on it to load the diff or follow a link to it.

chasenlehara commented on docs/guides/contributing.md:
```diff
@@ -0,0 +1,429 @@
+
+- “help wanted” for any issues with which the core team would like help
+- “easy” for issues the core team thinks are good for someone who’s new to contributing
+- “documentation” for issues related to documenting the APIs
```
Or maybe we should use “docs”?

justinbmeyer replied:
documentation is fine

justinbmeyer commented on docs/guides/contributing.md:
Maybe start with `DoneJS`? I know you are trying to be alphabetic.  But it's ok to ask any StealJS / CanJS question in the DoneJS gitter room.

chasenlehara replied:
I like that idea. I debated whether to include the other projects here… I think ideally we would link to their guides so this one doesn’t have to be updated, but since we don’t have all of this info in those guides, for now I like putting DoneJS on top.

justinbmeyer commented on docs/guides/contributing.md:
this isn't accurate with @m-mujica 's changes

justinbmeyer replied:
actually it probably is still

chasenlehara replied:
[I think it is](https://github.com/donejs/donejs/blob/major/package.json#L11), but if it’s not, I’ll update it when I submit a PR to merge `master` to `major`.

chasenlehara committed:
Update contribution guide

Based on feedback: https://github.com/donejs/donejs/pull/815#discussion_r101437717

chasenlehara committed:
Add some talks to the presentations list

Closes https://github.com/bitovi/bitovi/issues/11
